_posts/2021-08-02-changelog-152.markdown

```diff
@@ -0,0 +1,206 @@
+    generics, generates implementations of trait members and allows you to replace trait generics.
+
+  * [#6838] `Convert function to closure` to convert a local named function to closure (by [@nschoellhorn])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/convert-function-to-closure" w="600px" %}
+
+  * [#6813] `Convert closure to function` to convert a lambda expression to a local named function (by [@nschoellhorn])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/convert-closure-to-function" w="600px" %}
+
+  * [#6414] `Replace successive '_' with '..'` to replace redundant wildcards with a single `..` (by [@zeroeightysix])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/replace-wildcards" w="600px" %}
+
+  * [#7559] `Simplify dependency specification` to ...TODO inline tables in `Cargo.toml` dependencies
+
+    {% include gif-img.html path="/assets/posts/changelog-152/simplify-dependency-specification" w="600px" %}
+
+* Completion improvements:
+
+  * [#7487], [#7551], [#7536] Add completion for lambdas in expression positions that expect a function (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/lambda-completion" w="600px" %}
+
+  * [#7481] Add completion for `pub(crate)` and `pub()` visibility modifers (by [@Kobzol])
+
+  * [#6410] Add completion for layouts in `#[repr()]` attributes (by [@zeroeightysix])
+
+  * [#7534], [#7571] Improve completion of items inside impl trait (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/completion-member" w="600px" %}
+
+* [#7532] Add quick fix to implement missing supertrait(s) for a trait impl block (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-152/implement-supertraits" w="600px" %}
+
+* [#7492] Add a quick fix to remove an unused import (by [@Kobzol])
+
+* [#7479] TY&RES: support const arguments that looks like type arguments
+
+* [#7519] Add postfix template named `.wrap` that wraps an existing type with another type (by [@Kobzol])
+
+* [#7499] added a new filter to the Structure View panel, which toggles whether macro-generated items
+  should be included in the Structure View (by [@afetisov])
+
+* [#7495] The import optimizer will now remove unused imports (by [@Kobzol])
+
+* [#7321] `show macro expansion` actions and intentions now show a reason of the expansion failure
+
+* [#7308] Provide an experimental feature to add `--nocapture` flag to `cargo test` command
+
+* [#7228] Make links that look like Rust identifiers or paths (e.g., A, A::B, A::B::C) work correctly in
+  quick documentation by prepending `psi_element://` to them. It fixes #7209
+  Now clickable links inside stdlib docs should work properly (by [@shocoman])
+
+* [#6003] Allow inlining functions (using `Inline Function` refactoring) with diverging exit points
+  (like `panic!()`) (by [@SaarYogev])
+
+* [#7563] Add option to enable unused import inspection in doctests
```

Comment:
Let's group `Unused import` improvements together and mention that it is disabled by default. I'd place this group right after `Completion improvements`
```
* `Unused import` inspection improvements:
  * [#7492] Provide quick-fix to remove an unused import (by [@Kobzol])

  * [#7495] Remove unused imports by `Optimize Imports` action (`Code | Optimize Imports` or <kbd>Alt</kbd>+<kbd>Cmd</kbd>+<kbd>O</kbd>) (by [@Kobzol])

  * [#7563] Add option to enable unused import inspection in [doctests](https://doc.rust-lang.org/rustdoc/documentation-tests.html)

  Note, this is disabled by default for now due to possible false-positive warnings. You can enable it via `Preferences | Editor | Inspections` [settings](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#inspection-settings)